parser_test.go

```diff
@@ -3849,8 +3866,8 @@ func (s *testParserSuite) TestTablePartitionNameList(c *C) {
 		tableName, ok := source.Source.(*ast.TableName)
 		c.Assert(ok, IsTrue)
 		c.Assert(len(tableName.PartitionNames), Equals, 2)
-		c.Assert(tableName.PartitionNames[0], Equals, model.CIStr{"p0", "p0"})
-		c.Assert(tableName.PartitionNames[1], Equals, model.CIStr{"p1", "p1"})
+		c.Assert(tableName.PartitionNames[0], Equals, model.CIStr{O: "p0", L: "p0"})
```

Comment:
How about `c.Assert(tableName.PartitionNames[0].L, Equals, "p0"})`?

Reply:
This line is not a part of this PR. The go-lint in IDE told me it should be changed, so I fixed the lint.